### PR TITLE
Fixed the #10710 LightSensor documentation bug

### LINKED ISSUE #10710: LightSensor documentation example code bug
## On which page?
https://platform.uno/docs/articles/features/lightsensor.html

## What's wrong?
The Examples code `var LightSensor` starts with capital letter, so it's then taken as a local variable rather than the class.
OnPaste 20221210-171147

## Any feedback?
Solution: Use camelCase as in the [Barometer](https://platform.uno/docs/articles/features/barometer.html) example.

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): closes #10710

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

The Examples code throws `Cannot use local variable 'LightSensor' before it is declared.`

## What is the new behavior?

Used camelCase formatting on the `var LightSensor`, so the code doesn't throw the error.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]
👍 Thanks! 🚀

**Comment by ajpinedam** [comment]
Thanks for the fix!

Could you please update the commit message to match [the convention](https://www.conventionalcommits.org)?

TL;DR: in your case, it'd be something like:

```bash
docs: fixed the unoplatform#10710 LightSensor documentation bug 
```

**Review by ajpinedam — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@jeromelaban it seems the conventional commits and WIP validations are not completing for some reason 🤔
